src/librustdoc/passes/collect_intra_doc_links.rs

```diff
@@ -201,7 +204,7 @@ struct LinkCollector<'a, 'tcx> {
     /// See the code for associated items on inherent impls for details.
     kind_side_channel: Cell<Option<(DefKind, DefId)>>,
     /// Cache the resolved links so we can avoid resolving (and emitting errors for) the same link
```

Comment:
```suggestion
    /// Cache the resolved links so we can avoid resolving (and emitting errors for) the same link.
    /// The link will be `None` if it could not be resolved (i.e. the error was cached).
```